cross/ffmpeg-2.2.1/Makefile

```diff
@@ -0,0 +1,32 @@
+PKG_NAME = ffmpeg
+PKG_VERS = 2.2.1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://www.ffmpeg.org/releases/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/x264 cross/lame
+
+HOMEPAGE = http://www.ffmpeg.org/
+COMMENT  = FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video. It includes libavcodec - the leading audio/video codec library
+LICENSE  =
+
+REQUIRED_DSM = 5.1
```

Comment:
@cytec Noticed this while compiling, is there an issue with older toolchains? We don't do this with the 'other' ffmpeg.